Source/Classes/PINURLSessionManager.m

```diff
@@ -209,9 +209,18 @@ - (BOOL)responseRecoverableFrom404:(NSHTTPURLResponse*)response
 #if DEBUG
 - (void)concurrentDownloads:(void (^_Nullable)(NSUInteger concurrentDownloads))concurrentDownloadsCompletion
 {
-    [self.session getAllTasksWithCompletionHandler:^(NSArray<__kindof NSURLSessionTask *> * _Nonnull tasks) {
-        concurrentDownloadsCompletion(tasks.count);
-    }];
+    if (@available(macos 10.11, iOS 9.0, watchOS 2.0, tvOS 9.0,  *)) {
```

Comment:
@garrettmoon I remember we talked about not using `@available` to support Xcode < 9. Is this still the case?

Reply:
@maicki yeah, still shooting for that, but it'll be on top of this and will be a macro which uses `@availabile` on Xcode 9.